qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -78,14 +81,16 @@ def run(
             raise IBMQJobManagerInvalidStateError("Jobs were already submitted.")
 
         self._backend = backend
+        self._tags = job_tags.copy()
```

Comment:
If `job_tags` is not specified, it would default to `None` and this would fail. We should do a check before copying `job_tags` to prevent this